tachyon/crypto/sumcheck/multilinear/proof_keys_unittest.cc

```diff
@@ -0,0 +1,26 @@
+#include "tachyon/crypto/sumcheck/multilinear/proof_keys.h"
+
+#include <vector>
+
+#include "gtest/gtest.h"
+
+namespace tachyon::base {
+
+TEST(VerifyingKey, Copyable) {
+  crypto::sumcheck::VerifyingKey expected =
+      crypto::sumcheck::VerifyingKey::Random();
+
+  std::vector<uint8_t> vec;
+  vec.resize(base::EstimateSize(expected));
+  base::Buffer write_buf(vec.data(), vec.size());
+  ASSERT_TRUE(write_buf.Write(expected));
```

Comment:
Please apply the recent changed style! See https://github.com/kroma-network/tachyon/pull/300/commits/89980eb533999791d1ea5a00d3620e734ba523a7